Update instruction copy

ClementChaumel opened the pull request:
## Done

- Update instruction copy for 16.04 and 18.04 according to https://github.com/canonical-web-and-design/ubuntu.com/issues/9517#issuecomment-820183733

## QA

-  https://ubuntu-com-9669.demos.haus/advantage?test_backend=true


## Issue / Card

Fixes #9517

## Screenshots

![image](https://user-images.githubusercontent.com/11927929/114892893-7505a600-9e0d-11eb-858d-91e61a413ca5.png)

webteam-app commented:
Demo starting at https://ubuntu-com-9669.demos.haus

ClementChaumel commented:
@lechsandecki here are the changes to the instructions. Please review it and provide instruction for the 16.04 and 18.04 changes thanks!

lechsandecki commented on templates/advantage/table/_attached-machines.html:
```diff
@@ -3,32 +3,49 @@
-          <h5>Welcome to your new subscription</h5>
-          <p class="u-no-margin--bottom">If you’re using Ubuntu 14.04 ESM or 20.04 LTS, first attach each machine.</p>
+        <div class="advantage-table-section">
+            <h5>Welcome to your new subscription</h5>
+            <p class="u-no-margin--bottom">If you’re using Ubuntu 16.04 or 18.04 LTS you may need to install the UA client first.</p>
+            <div class="advantage-table-instruction">
+              <span>Add the PPA</span>
+              <code>sudo add-apt-repository ppa:ua-client/stable<br/>
+              sudo apt-get update</code>
+              <hr />
+              <span>Install the UA client</span>
+            <code>sudo apt install ubuntu-advantage-tools</code>
+            </div>
+          </div>
+        <div class="advantage-table-section">
+          <p class="u-no-margin--bottom">Once installed or if you’re using Ubuntu 14.04 ESM or 20.04 LTS, first attach each machine.</p>
```
I'd drop that. I suggest that, on this page, we keep the following:

to attach a machine: sudo ua abc
to detach a machine: sudo ua def

Please ensure that you have the latest version of the ua client installed: apt-get upgrade

lechsandecki commented on templates/advantage/table/_attached-machines.html:
```diff
@@ -3,32 +3,49 @@
-      <div class="row">
         <div class="advantage-table-section">
-          <h5>Ubuntu 14.04 ESM and 20.04 LTS</h5>
+            <p class="u-no-margin--bottom">If you’re using Ubuntu 16.04 or 18.04 LTS you may need to install the UA client first.</p>
+            <div class="advantage-table-instruction">
+              <span>Add the PPA</span>
+              <code>sudo add-apt-repository ppa:ua-client/stable<br/>
+              sudo apt-get update</code>
+              <hr />
+              <span>Install the UA client</span>
+            <code>sudo apt install ubuntu-advantage-tools</code>
+            </div>
+          </div>
+        <div class="advantage-table-section">
+          <p class="u-no-margin--bottom">Once installed or if you’re using Ubuntu 14.04 ESM or 20.04 LTS, first attach each machine.</p>
```
we also need to update the ESM screen (it's now 14.04 or 16.04, not just 14.04)
Similarly, Livepach can get activated via the ua client for each Ubuntu LTS now. We don't need separate "Ubuntu 16.04 LTS and 18.04 LTS" instructions anymore

albertkol commented:
@ClementChaumel you broke the table

![image](https://user-images.githubusercontent.com/6387619/116989651-ec6e7d00-acc9-11eb-8bcb-f8cfdddc0333.png)

ClementChaumel commented:
I've updated the copy according to your review @lechsandecki, please take another look:
![image](https://user-images.githubusercontent.com/11927929/117112198-002ce880-ad89-11eb-953f-7909b47e2326.png)
![image](https://user-images.githubusercontent.com/11927929/117112241-0de26e00-ad89-11eb-90b9-bed2e5be7c38.png)
![image](https://user-images.githubusercontent.com/11927929/117112268-189d0300-ad89-11eb-857b-7923d9ae8146.png)

and I fixed the table @albertkol :angel:

lechsandecki commented:
+1, looks great. Can we make it live asap, please?

SirSamTumless commented:
Hi @lsandecki I don't think it to say "we are slow implementation" in the public channnel. We ask for your review yesterday and you've only just come abck.
